src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,44 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import cn from 'classnames';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+type Props = {
+  selectedTodo: Todo | null;
```

Comment:
```suggestion
  selectedTodo: Todo;
```
You don't need to render modal without todo